src/reducers/content/meta.test.js

```diff
@@ -1,10 +1,22 @@
 import reducer from './meta';
+import * as t from '@/actions/types';
 
-describe('reducers/meta', () => {
-    it('should return the initial state', () => {
-        const expectedState = [];
-        const state = reducer();
+describe('reducers/content/meta', () => {
+  it('should return the initial state', () => {
+    const expectedState = [];
+    const state = reducer();
 
-        expect(state).toEqual(expectedState);
-    });
+    expect(state).toEqual(expectedState);
+  });
+
+  it('should set the given initial content', () => {
+    const action = {
+      type: t.SET_INITIAL_CONTENT,
+      payload: { meta: ['testContent1', 'testContent2'] },
+    };
+    const expectedState = ['testContent1', 'testContent2'];
+    const state = reducer(null, action);
```

Comment:
The default values (initialState here) are only applied if the value is `undefined`:
```
const state = reducer(undefined, action);
```

Reply:
good to know!